src/PerfView/Extensibility.cs

```diff
@@ -3620,7 +3621,9 @@ public static MutableTraceEventStackSource ThreadTimeStacks(this TraceLog eventL
         var stackSource = new MutableTraceEventStackSource(eventLog);
         stackSource.ShowUnknownAddresses = App.CommandLineArgs.ShowUnknownAddresses;
 
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
My preference would be to replace all such `[Obsolete]` items with one of the following:

1. Use [`<preliminary/>`](http://ewsoftware.github.io/XMLCommentsGuide/html/c16bece7-694e-48ca-802d-cf3ae9205c55.htm) in a documentation comment (requires maintenance and attention to assembly versioning)
1. Do nothing

If we keep using `[Obsolete]` for this manner, which it clearly wasn't designed for, my preference would be to go with the approach used in this pull request since technically that is adhering precisely to what was requested in the first place. The options with wrappers (3 and 4 in list) aren't really options because then we're letting the fact that something is new influence the design in ways that will carry over even when it's not new. That either increases design debt over time (if the wrappers stay) or increases our risk of breaking something during stabilization (if wrappers are removed).